src/walletConnect/saga.test.ts

```diff
@@ -570,6 +572,64 @@ describe('showActionRequest', () => {
       preparedTransaction: mockPreparedTransactions.transactions[0],
       prepareTransactionErrorMessage: undefined,
     })
+    expect(ValoraAnalytics.track).toHaveBeenCalledTimes(1)
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(
+      WalletConnectEvents.wc_request_propose,
+      expect.anything()
```

Comment:
Is this because we could have an array of any number of transactions?

Reply:
this was just me being lazy, it wasn't super obvious what the expected props were, there were too many 😄